src/components/Layout/NavBar/NavBar.tsx

```diff
@@ -6,8 +6,20 @@ import Button from '@mui/material/Button';
 import IconButton from '@mui/material/IconButton';
 import { routeMap } from 'router';
 import styles from './NavBar.styles';
-
-const navItems = ['Mis datos', 'Mis Mascotas', 'Reportar mascota'];
+const items = [
+	{
+		title: 'Mis datos',
+		path: routeMap.PROFILE.path,
+	},
+	{
+		title: 'Mis Mascotas',
+		path: routeMap.MY_PETS.path,
+	},
+	{
+		title: 'Reportar mascota',
+		path: routeMap.REPORT.path,
+	},
+];
```

Comment:
Good job! Could we move it to a general file?

For instance, a constants.ts file int he root path.

So, you should create a `constants.ts` file (in the root directory)

```suggestion
const NavBarItems = [
	{
		title: 'Mis datos',
		path: routeMap.PROFILE.path,
	},
	{
		title: 'Mis Mascotas',
		path: routeMap.MY_PETS.path,
	},
	{
		title: 'Reportar mascota',
		path: routeMap.REPORT.path,
	},
];
```